api/data_refinery_api/views.py

```diff
@@ -1174,39 +1172,28 @@ def _get_job_details(self, job):
         
         return name_match.group('type'), name_match.group('volume_id')
 
-    def _get_nomad_jobs(self):
-        """Calls nomad service and return all jobs"""
-        try:
-            nomad_host = get_env_variable("NOMAD_HOST")
-            nomad_port = get_env_variable("NOMAD_PORT", "4646")
-            nomad_client = nomad.Nomad(nomad_host, port=int(nomad_port), timeout=30)
-            return nomad_client.jobs.get_jobs()
-
-        except nomad.api.exceptions.BaseNomadException:
-            # Nomad is not available right now
-            return []
+    def _get_job_type(self, job): return self._get_job_details(job)[0]
```

Comment:
I don't like these

Reply:
Do you think it's better to have it as a variable with a lambda function?

```
get_job_type = lambda job: self._get_job_details(job)[0]
```